### ISSUE TITLE
Ability to see diff when in :G mode

### ISSUE THREAD

**Issue by simondrake** [tpope/vim-fugitive#2101]
**TL;DR**
===============
Is it possible, when in `:G` mode, to view the diff of a single line?

**Example**
===============

Say I have a folder, initialised with git, that just has a `README.md` file. If I then make a change to that file, I can

* View the diff from the command line

```
--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-testing
+testing testing one two three
```

* Open the file in (neo)vim, run `:G`, hover over the file in the bottom pane and hit `=` to see the same diff of the file

![image](https://user-images.githubusercontent.com/62928647/212071954-456ef2c7-2b42-4cd3-b0ba-7f9a94ac1b57.png)

* Open the file in (neo)vim, run `:G` and see which lines have been added/modified/deleted (same screenshot as above).

What I want to do, is put my cursor on line one and press something similar to `=` and see what the diff is of that line only. When a file only has a small amount of changes, doing it in the bottom pane is great, but when there's a load of changes it can be useful to see just one lines diff.

Is that currently possible with vim-fugitive?

**Comment by tpope** [comment]
Not sure I understand what you're asking for, but if you `:setl foldmethod=syntax`, you can collapse and expand individual hunks using folding. I think that's the closest thing Fugitive has to your request.
